lib/elixir/test/elixir/exception_test.exs

```diff
@@ -727,6 +727,17 @@ defmodule ExceptionTest do
 
       assert %UndefinedFunctionError{module: nil, function: :bar, arity: 0}
              |> message == "function nil.bar/0 is undefined"
+
+      defmodule(Behaviour, do: @callback(required() :: :ok))
```

Comment:
Let's define it as an optional callback too, so we don't get compilation warnings. :) I would also define a separate test, instead of changing an existing one.

Reply:
That's a great idea. Done.